File: js_project/Strings/ShoutClassifier.js
let s1 = " Abhi 2026!";
let s2 = "";

for (let i = 0; i < s1.length; i++) {
    let ch = s1.charAt(i);
    
    if (ch >= 'a' && ch <= 'z') {
        // Look up code, subtract 32, and build character out of it
        let upperCh = String.fromCharCode(ch.charCodeAt(0) - 32);
        s2 = s2 + upperCh;
    } else {
        s2 = s2 + ch;
    }
}

console.log("Original Input  : \"" + s1 + "\"");
console.log("Shout Conversion: \"" + s2 + "\"");

File: java_project/Strings/ShoutClassifier.java
package String;

public class ShoutClassifier {
    public static void main(String[] args) {
        String s1 = " Abhi 2026!";
        String s2 = "";
        for (int i = 0; i<s1.length(); i++){
            char ch = s1.charAt(i);
            if (ch >= 'a' && ch <= 'z'){
                char upperCh = (char) (ch - 32);
                s2 = s2 + upperCh;
            }
            else{
                s2 = s2+ch;
            }
            
        }
        System.out.println("Original Input  : \"" + s1 + "\"");
        System.out.println("Shout Conversion: \"" + s2 + "\"");
    }
}
